### PR TITLE
Add Tests to Verify NVCC Version in Minimal Cuda Image(ODS-483)

### PR DESCRIPTION
**Pull request by prudhvi-165**
Signed-off-by: prudhvi <pnammi@redhat.com>

### CODE REVIEW

File: tests/Tests/500__jupyterhub/minimal-cuda-test.robot [hunk 1]
```diff
@@ -45,6 +45,14 @@ Verify Tensorflow Library Can See GPUs In Minimal CUDA
     ...     ODS-1143
     Verify Tensorflow Can See GPU    install=True
 
+Verify Cuda Image Have NVCC Installed
+    [Documentation]     Verifies NVCC Version in Minimal CUDA Image
+    [Tags]  Sanity
+    ...     ODS-483
+    ${nvcc_version} =  Run Cell And Get Output    input=!nvcc --version| grep "release"| awk '{print $5}'
+    ${nvcc_version} =  Fetch From Left    ${nvcc_version}  ,
+    Should Be Equal As Strings    ${nvcc_version}  ${EXPECTED_CUDA_VERSION}
```
Comment (lugi0) [inline tests/Tests/500__jupyterhub/minimal-cuda-test.robot:54]: Is this actually checking the nvcc version or just the CUDA version?
If it's checking the CUDA version (given the comparison is done against `${EXPECTED_CUDA_VERSION}`), this is already done in `Verify Installed CUDA Version` in the `Verify CUDA Image Includes Expected CUDA Version` TC

Comment (prudhvi-165) [reply tests/Tests/500__jupyterhub/minimal-cuda-test.robot:54]: It must check if nvcc is installed or not.

**Comment by prudhvi-165** [comment]
Jenkins Link: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/656/
Result: Passed

**Review by bdattoma — APPROVED** [review]
lgtm

**Review by jgarciao — APPROVED** [review]
lgtm

**Pull request merged**